### DISCUSSION TITLE
missing options?

### DISCUSSION THREAD

**Discussion by gsgoldma** [Coyote-A/ultimate-upscale-for-automatic1111#43] [Q&A]
![missing options](https://user-images.githubusercontent.com/113954966/215852439-9444b707-bf4e-4ea0-8941-b89180695c3b.png)

am I wrong or should there be additional settings there?

**Comment by Jopezzia** [comment]
Yes, looks like you have some bug with installation. Just remove it from extensions folder. If it's still relevant
